ca/ca.go

```diff
@@ -438,16 +438,23 @@ func (ca *CAImpl) CompleteOrder(order *core.Order) {
 	order.Unlock()
 }
 
-func (ca *CAImpl) GetSubjectKeyIDs() core.SubjectKeyIDs {
-	skis := core.SubjectKeyIDs{}
+// GetIntermediateBySKID attempts to match the incoming Authority Key Idenfitier
+// (AKID) bytes to the Subject Key Identifier of an intermediate certificate. It
+// returns an error if no match is found.
+func (ca *CAImpl) GetIntermediateBySKID(issuer []byte) error {
```

Comment:
Either have this return the intermediate if it finds one, or change the name to something like "RecognizedSKID"